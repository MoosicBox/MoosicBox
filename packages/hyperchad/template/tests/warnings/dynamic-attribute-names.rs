use hyperchad_template::container;

fn main() {
    let name = "href";
    container! {
        a (name)="about:blank" {}
    };
}
