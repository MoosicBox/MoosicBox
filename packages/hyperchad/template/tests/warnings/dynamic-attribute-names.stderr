error: expected one of: identifier, literal, `.`, `#`, curly braces, `;`
 --> tests/warnings/dynamic-attribute-names.rs:5:5
  |
5 | /     container! {
6 | |         div (name)="about:blank" {}
7 | |     };
  | |_____^
  |
  = note: this error originates in the macro `container` (in Nightly builds, run with -Z macro-backtrace for more info)

warning: unused variable: `name`
 --> tests/warnings/dynamic-attribute-names.rs:4:9
  |
4 |     let name = "href";
  |         ^^^^ help: if this is intentional, prefix it with an underscore: `_name`
  |
  = note: `#[warn(unused_variables)]` (part of `#[warn(unused)]`) on by default
